### DISCUSSION TITLE
nuclei-plus_v7.5.11

### DISCUSSION THREAD

**Discussion by G3G4X5X6** [Yong-An-Dang/nuclei-plus#38] [Announcements]
### 🐛 修复
-  #36 修复模板列表序号排序错乱问题。
-  #37 修复配置保存读取编码可能不一致的问题。

**Full Changelog**: https://github.com/Yong-An-Dang/nuclei-plus/compare/v7.5.9...v7.5.11

<hr /><em>This discussion was created from the release <a href='https://github.com/Yong-An-Dang/nuclei-plus/releases/tag/v7.5.11'>nuclei-plus_v7.5.11</a>.</em>

**Comment by royalaholddelhaize1** [comment]
How to change language to English?

**Reply by G3G4X5X6** [reply]
> How to change language to English?

```
# 国际化，指定本地语言（为空跟随系统设置）：zh_CN, en_US
nuclei.locale=en_US
```

![image](https://github.com/user-attachments/assets/5990dde9-e42c-431e-baa2-a37839a56ba9)
